annex/annex.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -ex
+
+MONTAGU_BASENAME=`basename ${PWD}`
+MONTAGU_NETWORK="${MONTAGU_BASENAME}_default"
+
+MONTAGU_DB="${MONTAGU_BASENAME}_db_1"
+MONTAGU_DB_ANNEX="${MONTAGU_BASENAME}_db-annex_1"
+
+GIT_ID=$(git rev-parse --short HEAD)
+REGISTRY=docker.montagu.dide.ic.ac.uk:5000
+MIGRATE_IMAGE=$REGISTRY/montagu-migrate:$GIT_ID
+
+docker pull $MIGRATE_IMAGE
+docker pull $REGISTRY/montagu-db:i880
+
+function cleanup {
+    set +e
+    docker-compose stop
+    docker-compose rm -f -v
+    docker network rm $MONTAGU_NETWORK
+    docker volume rm "${MONTAGU_BASENAME}_db_volume" "${MONTAGU_BASENAME}_db_annex_volume"
+trap cleanup EXIT
+
+MONTAGU_DB_VERSION=i880 docker-compose up -d
```

Comment:
Is this just missing a new line?

Reply:
I don't think so - what looks missing to you?

Reply:
So are you assigning the text `i880 docker-compose up -d` to `MONTAGU_DB_VERSION`? Are you assigning the variable and then running docker-compose? What's going on here?

Reply:
This sets the variable `MONTAGU_DB_VERSION` to the value `i880` for the next command; the docker compose file references it (following the main montagu docker compose file)